library/Fission/Models.hs

```diff
@@ -54,6 +54,7 @@ HerokuAddOn
 
 User
   publicKey     Key.Public    Maybe
+  exchangeKeys  [Key.Public]  Maybe
```

Comment:
From a purely information perspective

`Maybe [a] ~ [a]`

Reply:
hmm yeah good point :thinking: 

If we have a required col in the Model will it automatically update every row in the DB to have an empty array or will we need to do that manually?

Reply:
You can set `default=` in the models [Reference](https://github.com/yesodweb/persistent/blob/master/docs/Persistent-entity-syntax.md#change-tablecollection-key-definition-field-name-andor-type-persistent--21)

Probably not needed here, but it's also possible to do a custom translation to/from the database with typeclasses [PersistField](https://www.stackage.org/haddock/lts-16.2/persistent-2.10.5.2/Database-Persist-Class.html#g:6) and friends